api/src/main/java/io/minio/MinioClient.java

```diff
@@ -4163,9 +4163,7 @@ private void putObject(
       headerMap.putAll(options.sse().headers());
     }
 
-    if (headerMap.get("Content-Type") == null) {
-      headerMap.put("Content-Type", options.contentType());
-    }
+    headerMap.put("Content-Type", options.contentType());
```

Comment:
`Content-Type` is passed as part of `options' header` will be overwritten by this. Is this the expected behavior?

Reply:
Please check `PutObjectOptions.contentType()` method for precedence.